scripts/eia/doi_Chinyanja-Solidaridad-Soy-NOT.R

```diff
@@ -17,22 +17,24 @@ carob_script <- function(path) {
    uri <- "doi:Chinyanja-Solidaridad-Soy-NOT"
    group <- "eia"
    
-   dset <- data.frame(
-      # Need to fill-in metadata...
+   meta <- data.frame(
       # carobiner::read_metadata(uri, path, group, major=2, minor=0),
       uri = carobiner::simple_uri(uri),
       dataset_id = uri,
-      authors =NA,
+      authors ="Mary Jane;John Doe",
+      publication= NA,
       data_institute =NA,
       title = NA,
       group = group,
-      license = 'Some license here...',
+      license = NA,
       carob_contributor = 'Cedric Ngakou',
       project = 'Excellence in Agronomy - Chinyanja-Solidaridad-Soy-NOT',
```

Comment:
This should be just 'Excellence in Agronomy', since we now have the two other fields (`use_case` and `activity`)